tests/e2e/device-plugin_test.go

```diff
@@ -68,7 +64,7 @@ var _ = NVDescribe("GPU Device Plugin", func() {
 			"nodes",
 			"namespaces",
 			"deployments",
-			"demonsets",
+			"daemonsets",
```

Comment:
This makes me things we should set this as the default in the collector options. Out of scope for this PR though.